### ISSUE TITLE
How to update hubble version running on a dedicated machine

### ISSUE THREAD

**Issue by talktopiyush** [Autodesk/hubble#140]
Hello there,

How to update hubble version running as service on a dedicated machine. I am not able to find the right documentation.

Regards
Piyush

**Comment by pluehne** [comment]
If you run Hubble on a dedicated machine as detailed in the [setup instructions](https://github.com/Autodesk/hubble/blob/82015319d8aa36ae9790d201b25ec34155d015ec/updater/README.md#setup-on-a-dedicated-machine), you’ll have cloned this Git repository in step 3.

To update Hubble to the latest release, just check out the newest release tag, currently v0.2.0:

```sh
$ git checkout v0.2.0
```

To get the newest development version in the `master` branch (we don’t recommend this though), perform the following steps

```sh
$ git checkout master
$ git pull origin master
```

If you need further assistance, please let me know.

PS: As stated earlier, we recommend using the latest release tag. As you’re already on the `master` branch and have experienced issues with that, please try the newest `master` commit for now but consider switching to release tags when we make the next release (which will probably be 0.2.1).

**Comment by talktopiyush** [comment]
thanks, looks like I am missing something?
`# git checkout v0.2.0`
`error: pathspec 'v0.2.0' did not match any file(s) known to git.`

I think we need to do a git fetch before? isn't it.

**Comment by talktopiyush** [comment]
`# git fetch origin`
`error:  while accessing https://github.com/autodesk/hubble/info/refs`
`fatal: HTTP request failed`
`

**Comment by pluehne** [comment]
Yeah, you need fo fetch first to get the info about the newest changes.

I’m not sure why you’re getting HTTP request errors though. Could it just be a network issue?
